src/main/java/com/suse/salt/netapi/datatypes/Batch.java

```diff
@@ -8,16 +12,43 @@
 public class Batch {
     // The actual batch string
     private String batch;
+    private Optional<Double> delay = Optional.empty();
 
     private Batch(String batch) {
         this.batch = batch;
     }
 
+    private Batch(String batch, Double delay) {
+        this.batch = batch;
+        this.delay = Optional.of(delay);
+    }
+
+    public Batch delayed(Double seconds) {
```

Comment:
Here it might be really good to have some documentation about the `delayed` config parameter.